### DISCUSSION TITLE
Command PhaseScriptExecution failed with a nonzero exit code

### DISCUSSION THREAD

**Discussion by swantanbarua** [mozilla-mobile/firefox-ios#14731] [Q&A] [answered]
Hello people. I'm a newbie in Open Source space and would want to contribute to this great project. 

I'm facing "Command PhaseScriptExecution failed with a nonzero exit code" error when I'm trying to run the project. 

Please help me out to resolve it. Thanks in advance.

**Comment by lmarceau** [comment]
Hello! It normally tells us more when this error happens, can you check to share more about it? But first, can you make sure you followed all instructions in the `Building the code` section in [our readme](https://github.com/mozilla-mobile/firefox-ios#building-the-code)?

**Reply by swantanbarua** [reply]
I followed all the steps from 1-8 provided in the README file. Then on running the app, build is failed with the aforementioned error.

**Reply by lmarceau** [reply]
Yes ok, but can you give more than `Command PhaseScriptExecution failed with a nonzero exit code`? It normally comes with a description on what failed.

**Reply by swantanbarua** [reply]
I'm providing a screenshot. Hopefully this helps. 
<img width="1118" alt="Screenshot 2023-06-06 at 12 07 23 AM" src="https://github.com/mozilla-mobile/firefox-ios/assets/85452778/05d7c96a-306c-4dd1-b19c-71056ed3065e">

**Reply by lmarceau** [reply]
Can you go to the report navigator, click on the build that failed, scroll to the bottom of message and send us that?

Example (but this build was green)
![Screenshot 2023-06-05 at 2 41 24 PM](https://github.com/mozilla-mobile/firefox-ios/assets/11338480/7c219bca-a011-43dd-bf07-92ad6bcc8b06)

**Reply by swantanbarua** [reply]
Yes! Here it is. 
<img width="1258" alt="Screenshot 2023-06-06 at 12 19 19 AM" src="https://github.com/mozilla-mobile/firefox-ios/assets/85452778/007711dd-eab9-404b-b9ae-b11a1259fe9f">

**Reply by swantanbarua** [reply]
I have a folder named "Open Source" and inside that I have the "firefox-iOS". Is it possible that the error is because of the whitespace in my folder name?
<img width="897" alt="Screenshot 2023-06-06 at 12 22 57 AM" src="https://github.com/mozilla-mobile/firefox-ios/assets/85452778/ad7bc35b-7755-4699-af19-0925224d2e1b">

**Reply by lmarceau** [reply, marked as answer]
Oh. Possibly! You can try renaming the folder to `Open-Source` and try if it fixes the problem?

**Reply by swantanbarua** [reply]
Hi @lmarceau I resolved the issue. Apparently it was because of a whitespace in my directory "Open Source". I changed it to "OpenSource" and placed "Firefox-iOS" in there. Now I'm able to run the project. Thanks for guiding. :)

**Reply by janbrasna** [reply]
This should be now fixed via https://github.com/mozilla-mobile/firefox-ios/pull/35000 💪
